charts/karmada/templates/karmada-scheduler.yaml

```diff
@@ -50,8 +50,8 @@ spec:
           command:
             - /bin/karmada-scheduler
             - --kubeconfig=/etc/kubeconfig
-            - --bind-address=0.0.0.0
-            - --secure-port=10351
+            - --metrics-bind-address=0.0.0.0:10351
+            - --health-probe-bind-address=0.0.0.0:10351
```

Comment:
Probably the same reason as https://github.com/karmada-io/karmada/pull/5290#issuecomment-2311992249. 
We have to revert charts changes to get CI pass.

Reply:
updated,Thanks!